src/main/java/io/opentracing/contrib/global/concurrent/TracedCallable.java

```diff
@@ -0,0 +1,68 @@
+package io.opentracing.contrib.global.concurrent;
+
+import com.google.common.io.Closer;
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer.SpanBuilder;
+import io.opentracing.contrib.global.GlobalTracer;
+
+import java.util.concurrent.Callable;
+
+/**
+ * Convenience {@link Callable} wrapper that will execute within a new {@link Span} if an
+ * {@link #withOperationName(String) operationName} is provided.<br>
+ * If no operationName is provided, the call will be executed without starting a new Span.
+ */
+public class TracedCallable<T> implements Callable<T> {
+
+    private final Callable<T> delegate;
+    private String operationName = "";
+    private SpanContext parentContext = null;
+
+    protected TracedCallable(Callable<T> delegate) {
+        if (delegate == null) throw new NullPointerException("Callable delegate is <null>.");
+        this.delegate = delegate;
+    }
+
+    public static <T> TracedCallable<T> of(Callable<T> delegate) {
+        return new TracedCallable<T>(delegate);
+    }
+
+    public TracedCallable<T> withOperationName(String operationName) {
+        this.operationName = operationName;
+        return this;
+    }
+
+    public TracedCallable<T> asChildOf(SpanContext parent) {
+        this.parentContext = parent;
+        return this;
+    }
+
+    protected Span startNewSpan() {
+        SpanBuilder spanBuilder = GlobalTracer.get().buildSpan(operationName);
+        if (parentContext != null) spanBuilder = spanBuilder.asChildOf(parentContext);
+        return spanBuilder.start();
+    }
```

Comment:
Agreed, the comment needs fixing. I'll also add an example to the javadoc. Passing in a span (from the scheduling thread) defeats the purpose as you could want to schedule a call and have it traced. So the call needs to start / finish the span itself. I removed all context propagation from this library, but you can still add a 'known' parent context by calling SpanBuilder-like methods on the TracedCallable object.
I'll change the javadoc.